draft-ietf-core-new-block.xml

```diff
@@ -859,9 +837,14 @@ missing-block-number = uint
     </section>
 
     <section anchor="cc" title="Congestion Control">
-      <t>The transmission of the payloads of a body either SHOULD all be
-      Confirmable or all be Non-confirmable. This is meant to simplify the
-      congestion control procedure.</t>
+      <t>The transmission of the payloads of a body over UDP or DTLS connection
+      SHOULD either all be Confirmable or all be Non-confirmable.
+      This is meant to simplify the congestion control procedure.</t>
+
+      <section anchor="cc-tcp" title="Reliable Transports">
```

Comment:
```suggestion
```

Reply:
Same comment about the usage of Unreliable as you can argue CoAP is a reliable transport when using UDP when using CON.  I agree that I do not particularly like adding in this section.